distributions/Dist_ams1396.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_ams1396(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -0.5
+		self.x_max = 0.5
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return (5*x^11)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return (.000244140625) 
```

Comment:
you should have an equation based on x_min and xmax where you calculate the std using calculus.